lib/kv-router/BENCH_SETUP.md

```
@@ -0,0 +1,234 @@
+|-----------------|--------------|----------------|-----|
+| 30,000 ms | 11,861 | 11,532 | 7,536 µs |
+| 18,455 ms | 19,281 | 18,046 | 13,362 µs |
+| 11,352 ms ⚠ | 31,345 | 25,240 | 11,076 µs |
+| 6,983 ms ⚠ | 50,956 | 30,547 | 10,529 µs |
+| 4,296 ms ⚠ | 82,827 | 25,050 | 11,607 µs |
+| 2,643 ms ⚠ | 134,629 | 23,684 | 10,233 µs |
+| 1,626 ms ⚠ | 218,834 | 27,543 | 11,143 µs |
+| 1,000 ms ⚠ | 355,824 | 25,249 | 12,388 µs |
+
+**Branch-sharded depth=2:**
+| Benchmark window | Offered ops/s | Achieved ops/s | p99 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)